modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/opt/GridH2Table.java

```diff
@@ -857,6 +873,67 @@ public boolean rebuildFromHashInProgress() {
         return commitUserIndex(ses, idxName);
     }
 
+    /**
+     * Checks and replace if provided column id matches value alias column.
+     *
+     * @param col Input column.
+     * @return Transformed result if needed.
+     */
+    private String aliasNameTransformer(IndexColumn col) {
```

Comment:
Method name is poor. It seems that name should be or should contain some verb that describe action.